src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import './Loader.scss';
```

Comment:
Make sure you are using the loader syntax instead of Webpack syntax in the imports in `webpack.config.js` for the `Loader.scss` file. 🚀